share/functions/__fish_pipestatus_with_signal.fish

```diff
@@ -0,0 +1,5 @@
+function __fish_pipestatus_with_signal --description "Print arguments from \$pipestatus replacing values with signal names where appropriate"
+    for pstat in $argv
+        echo (__fish_status_to_signal $pstat)
```

Comment:
This `echo` is basically useless. Just `__fish_status_to_signal $pstat` will already work.

Reply:
doh! of course, thank you. Pushed the change.